opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -254,7 +257,12 @@ namespace cpgrid
         std::cout << "Processing eclipse data." << std::endl;
 #endif
         processed_grid output;
-        process_grdecl(&input_data, z_tolerance, &output);
+        const size_t global_nc = input_data.dims[0] * input_data.dims[1] * input_data.dims[2];
+        std::vector<int> is_aquifer_cell(global_nc, 0);
```

Comment:
Could we set up `aquifer_cell_volumes_local`here and use that instead of `is_aquifer_cell` to judge?

Reply:
It looks like that we can not, because `aquifer_cell_volumes_local` needs indices after processed.